lib/HTTP/Tiny.pm

```diff
@@ -1792,10 +1805,15 @@ certificate has been verified by a CA. Assuming you trust the CA, this will
 protect against L<machine-in-the-middle
 attacks|http://en.wikipedia.org/wiki/Machine-in-the-middle_attack>.
 
-Certificate verification requires a file containing trusted CA certificates.
+Certificate verification requires a file or directory containing trusted CA
+certificates.
+
+If C<IO::Socket::SSL::default_ca()> is available, HTTP::Tiny will use it to
+detect the default location of your CA certificates. This function also supports
+the environment variables C<SSL_CERT_FILE> and C<SSL_CERT_DIR>, and will
+ultimately fail over to L<Mozilla::CA> if no certs are found.
```

Comment:
My understanding of the patch as it is could be wrong, but it first tests for a custom cert path provided by the user, then IO::Socket::SSL (if installed), then Mozilla::CA (if installed), then falls back to hardcoded paths from popular distros. If this is correct, then I think that order should be made more clear in the comments.

Reply:
I've added comments and updated docs to make this a bit more clearer

Reply:
Note: I've removed the failover to Mozilla::CA (if installed) in HTTP::Tiny by bumping the IO::Socket::SSL requiredment to 1.968 (which introduced default_ca())

Reply:
I think this should be resolved now?